src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -776,14 +776,29 @@ default int levelIntervalOverride() {
         return 0;
     }
 
+    @ConfigItem(
+        keyName = "xpInterval",
+        name = "Post-99 XP Interval",
+        description = "XP interval at which to fire notifications (in millions).<br/>" +
+            "Does not apply to skills that are below level 99<br/>" +
+            "Disabled if set to 0",
```

Comment:
should probably also say something about how it's independent from the `virtual levels` setting